File: Web_search_job/ClientApp/src/app/models/backend/dtos/profiles/resume-education.dto.ts
import {Industry} from "../other/industry.dto";
import {LocationDataDTO} from "../other/location-data.dto";
import {ActiveResumeSectionDTO} from "./active-resume-section.dto";

export interface ResumeEducationDTO {
  id: number | null;
  resumeId: number;
  educationProffesion: string;
  industryId: number | null;
  locationId: number | null;
  educationYear: number;
  industry: Industry | null;
  location: LocationDataDTO | null;
}

File: Web_search_job/DTO/Resume/ResumeEducationDTO.cs
using Web_search_job.DTO.Other;

namespace Web_search_job.DTO.Resume
{
    public class ResumeEducationDTO
    {
        public int? Id { get; set; }
        public int ResumeId { get; set; }
        public string EducationProffesion { get; set; }
        public int? IndustryId { get; set; }
        public int? LocationId { get; set; }
        public int EducationYear { get; set; }
        public IndustryDataDTO? Industry { get; set; }
        public LocationDataDTO? Location { get; set; }
    }
}
